docs/stargz-estargz.md

```diff
@@ -1,81 +1,223 @@
+  Empty means zero or unknown.
+  Otherwize, the value is in UTC RFC3339 format.
+
+- **`linkName`** *string*
+
+  This OPTIONAL property contains the link target of `symlink` and `hardlink`.
+
+- **`mode`** *int64*
+
+  This OPTIONAL property contains the permission and mode bits.
+
+- **`uid`** *int*
+
+  This OPTIONAL property contains the user ID of the owner of this file.
+
+- **`gid`** *int*
+
+  This OPTIONAL property contains the group ID of the owner of this file.
+
+- **`userName`** *string*
+
+  This OPTIONAL property contains the username of the owner.
+
+- **`groupName`** *string*
+
+  This OPTIONAL property contains the groupname of the owner.
+
+- **`offset`** *int64*
+
+  This OPTIONAL property contains the offset of the gzip header of the regular file or chunk in the archive.
+
+- **`devMajor`** *int*
+
+  This OPTIONAL property contains the major device number for character and block device files.
+
+- **`devMinor`** *int*
+
+  This OPTIONAL property contains the minor device number for character and block device files.
+
+- **`xattrs`** *string-bytes map*
+
+  This OPTIONAL property contains the extended attribute for the tar entry.
+
+- **`digest`** *string*
+
+  This OPTIONAL property contains the OCI [Digest](https://github.com/opencontainers/image-spec/blob/v1.0.1/descriptor.md#digests) of the regular file contents.
+  TOCEntries of non-empty `reg` file MUST set this property.
+
+- **`chunkOffset`** *int64*
+
+  This OPTIONAL property contains the offset of this chunk in the regular file payload.
+  Note that this is the offset of this chunk in the decompressed file content.
+  TOCEntries of `chunk` type MUST set this property.
+
+- **`chunkSize`** *int64*
+
+  This OPTIONAL property contains the decompressed size of this chunk.
+  The last `chunk` in a `reg` file or `reg` file that isn't chunked MUST set this property to zero.
+  Other `reg` and `chunk` MUST set this property.
+
+- **`chunkDigest`** *string*
+
+  This OPTIONAL property contains an OCI [Digest](https://github.com/opencontainers/image-spec/blob/v1.0.1/descriptor.md#digests) of this chunk.
+  TOCEntries of non-empty `reg` and `chunk` MUST set this property.
+  This MAY be used for verifying the data of this entry in the way described in [Content Verification in eStargz](/docs/verification.md).
+
+### Footer
+
+At the end of the archive, a *footer* MUST be appended.
+The structure of the footer is inherited from [stargz](https://github.com/google/crfs).
+This MUST be an empty gzip member ([RFC1952](https://tools.ietf.org/html/rfc1952)) whose [Extra field](https://tools.ietf.org/html/rfc1952#section-2.3.1.1) contains the offset of TOC in the archive.
+The footer MUST be the following 47 bytes (1 byte = 8 bits in gzip).
+
+```
+10 bytes  gzip header
+2  bytes  XLEN (length of Extra field) = 22 (16 hex digits + len("STARGZ"))
+22 bytes  Extra field (fmt.Sprintf("%016xSTARGZ", offsetOfTOC))
+5  bytes  flate header
+8  bytes  gzip footer
+(End of the eStargz blob)
```

Comment:
Is this section about plain stargz or eStargz?

Reply:
Fixed the footer section to make it compliant to [Extra field in RFC 1952](https://tools.ietf.org/html/rfc1952#section-2.3.1.1) (thus eStargz's footer).
The implementation will be the following-up PR.